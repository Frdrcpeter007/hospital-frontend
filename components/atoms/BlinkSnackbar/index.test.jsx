import { render } from "@testing-library/react";
import BlinkSnackbar from ".";

describe("BlinkSnackbar", () => {
  it("should render", () => {
    const { container } = render(<BlinkSnackbar />);
    expect(container).toMatchSnapshot();
  });
});
