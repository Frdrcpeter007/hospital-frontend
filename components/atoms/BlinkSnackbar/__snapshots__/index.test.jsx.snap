// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlinkSnackbar should render 1`] = `<div />`;
